docs/src/appendix/chisel3-vs-chisel2.md

```diff
@@ -16,7 +19,7 @@ modifications are:
    val wire = UInt(width = 15)
    ```
    becomes (in Chisel3):
-   ```scala
+   ```scala mdoc:compile-only
    val wire = Wire(UInt(15.W))
    ```
```

Comment:
````suggestion
```scala mdoc:compile-only
val wire = Wire(UInt(15.W))
```

I can't properly suggest this everywhere (including above), but it seems that mdoc doesn't work when the code block is tabbed over.
````

Reply:
😱 maybe that explains the flakiness in the results I was seeing

Reply:
ok, so you are sayign this is all over the place and I need to audit for it...

Reply:
Alternatively just leave them all as `scala` 🤷

Reply:
is there a reason we want them tabbed over? `compile-only` is better than nothing...

Reply:
I don't think there's any good reason to have them tabbed over, and as it seems there's a very bad reason in that mdoc doesn't support it

Reply:
k working on it